### DISCUSSION TITLE
Commit failed Committing the files to the repository failed. Cannot read properties of undefined (reading 'rule_run').

### DISCUSSION THREAD

**Discussion by saivamshi9414** [community/community#122308] [Repositories]
iam not able to upload files i got an error please help me with this out

**Comment by x-CHRIS-x** [comment]
same error i got now

**Comment by z3r0xxx** [comment]
I have same error

**Reply by marganov** [reply]
i have same problem right now(

**Comment by Avotexs** [comment]
i think you need to ensure that you're using the latest version of Git try to update Git and Before committing, run git status to check if all your files properly tracked by Git

**Reply by z3r0xxx** [reply]
its on git web version

**Comment by goldieowner17** [comment]
Hey @saivamshi9414! Thanks for being a part of the GitHub community. I've moved your conversation over to "General" as it doesn't quite fit with the subject area of Discover.

**Comment by salma-elbakkouri** [comment]
you need to ensure that you're using the latest version of Git

**Comment by JJCUBER** [comment]
This is occurring when uploading files directly through the github web UI.

**Reply by searchforhofni** [reply]
How can one fix it?

**Reply by JJCUBER** [reply]
Unfortunately, it mysteriously worked after trying another time or two, so I have no clue how to "fix" it.

**Comment by Rabbonos** [comment]
> Unfortunately, it mysteriously worked after trying another time or two, so I have no clue how to "fix" it.

Yep, I tried 2-3 times and it worked then. No idea why the problem ,though.

**Comment by jonathankto** [comment]
Hi my friend,
I myself had this bug, and after making 3 attempts, the bug resolved itself; So, you can also try several times, and make sure your connection is good.

**Comment by ffordian** [comment]
I also have this problem with the web UI, but it seems to only be with some larger commits. If I break the the same upload up into several smaller chunks it works.

**Comment by HK-Adventures** [comment]
i tried it several times but it's not comminting idk what is happeing

**Comment by hyungin0505** [comment]
Same issue occurred on web UI when I uploaded about 21Mb folder.
After I divided the folders and uploaded 3Mb folder, I uploaded the remaining 18Mb folder.
idk why, they have been uploaded without any problems.

**Reply by ghost** [reply]
I was getting the same error when uploading a file of just 4mb (85 files). Instead, I committed a part of the files and it worked.
Thanks to the guy who said so from the onset

**Comment by Sebastianfiet** [comment]
As some people have already pointed out, I think you just gotta keep trying, I got the same error while trying to upload a file, and after 4 attempts it finally let me do it.

**Comment by IfeelPa1n** [comment]
I only got 37 files got this error 3 times ;-;

**Comment by waghmareseema** [comment]
I also got same error but tried uploading 3 times on 4th time it got uploaded.
